alpha/external-web/app/routes/escorts.js

```diff
@@ -13,7 +13,7 @@ var client = require('../eligibility-client')
  */
 router.get('/escorts/:claimant_id', function (request, response) {
   var id = request.params.claimant_id
-  console.log('GET /about-you/' + id + ' called.')
+  console.log('GET /escorts/' + id + ' called.')
```

Comment:
are these log statements necessary? We get outputs of route access when debugging the app and if we wanted we could add log middleware to log details of the req